### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by m-rey**
* Remove trailing whitespace 
* Set consistent indentation
* Use pgrep to count running secbox instances

### CODE REVIEW

File: secbox [hunk 1]
```diff
@@ -241,13 +241,13 @@ suse_internal_network() {
 
 secbox_instances_using_nfs() {
     local _instances=0
-    _instances=$(ps -xo pid,cmd | grep -v grep | grep -v $$ | grep -E "secbox.* --nfs" | wc -l)
+    _instances=$(pgrep --count --full "secbox.* --nfs")
```
Comment (StayPirate) [inline secbox:244]: In order to make it works with `pgrep` the following check needs to be changed to `-le 3`. Can you please add this?

https://github.com/StayPirate/secbox/blob/983fb1db89ed372a3db1d5402f1d4422581082f5/secbox#L115

File: secbox [hunk 2]
```diff
@@ -241,13 +241,13 @@ suse_internal_network() {
...
     echo "$_instances"
 }
 
 secbox_instances_using_sshfs() {
     local _instances=0
-    _instances=$(ps -xo pid,cmd | grep -v grep | grep -v $$ | grep -E "secbox.* --sshfs" | wc -l)
+    _instances=$(pgrep --count --full "secbox.* --sshfs")
```
Comment (StayPirate) [inline secbox:250]: In order to make it works with `pgrep` the following check needs to be changed to `-le 3`. Can you please add this?

https://github.com/StayPirate/secbox/blob/983fb1db89ed372a3db1d5402f1d4422581082f5/secbox#L126

**Review by StayPirate — CHANGES_REQUESTED** [review]
Please check the two added comments, and rebase your branch on top of the latest master's commit. I'll then be able to merge this PR. Thanks for your contribution.

**Comment by m-rey** [comment]
done, please take a look

**Review by StayPirate — APPROVED** [review]

**Pull request merged**